independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Webex Integration",
+    "description": "WebEx API for Power Platform\n\n- Setup meeting from Power Platform",
+    "contact": {
+      "name": "UCL",
+      "email": "oscar.hui.21@ucl.ac.uk"
```

Comment:
@ocarhui can you please add "url" property in the contact section? it could be any page for the publisher (LinkedIn, GitHub...)